pymmcore_plus/core/_mmcore_plus.py

```diff
@@ -659,6 +659,62 @@ def getImage(self, *args, fix=True) -> np.ndarray:
         img = super().getImage(*args)
         return self._fix_image(img) if fix else img
 
+    def startContinuousSequenceAcquisition(self, intervalMs: float = 0) -> None:
+        """Start a ContinuousSequenceAcquisition."""
+        super().startContinuousSequenceAcquisition(intervalMs)
+        self.events.startContinuousSequenceAcquisition.emit()
+
+    @overload
+    def startSequenceAcquisition(
+        self,
+        numImages: int,
+        intervalMs: float,
+        stopOnOverflow: bool,
+    ) -> None:
+        ...
+
+    @overload
+    def startSequenceAcquisition(
+        self,
+        cameraLabel: str,
+        numImages: int,
+        intervalMs: float,
+        stopOnOverflow: bool,
+    ) -> None:
+        ...
+
+    def startSequenceAcquisition(self, *args) -> None:
+        if len(args) == 3:
+            numImages, intervalMs, stopOnOverflow = args
+            super().startSequenceAcquisition(numImages, intervalMs, stopOnOverflow)
+            cameraLabel = super().getCameraDevice()
+        else:
+            cameraLabel, numImages, intervalMs, stopOnOverflow = args
+            super().startSequenceAcquisition(
+                cameraLabel, numImages, intervalMs, stopOnOverflow
+            )

```

Comment:
I think this simplifies it a bit and allows the first thing that happens to be the C++ to check the arguments so we are garunteed to have either 3 or 4, previously that wasn't garunteed and you end up with an error when unpacking args in the else branch (at least I think...)

```suggestion
        super().startSequenceAcquisition(numImages, intervalMs, stopOnOverflow)
        if len(args) == 3:
            numImages, intervalMs, stopOnOverflow = args
            cameraLabel = super().getCameraDevice()
        else:
            cameraLabel, numImages, intervalMs, stopOnOverflow = args
```

Reply:
woops sorry this sholda been `super().startSequenceAcquisition(*args)`


and same deal with stop acquisition